holisticai/bias/metrics/_regression.py

```diff
@@ -7,6 +7,47 @@
 from ...utils._validation import _check_non_empty, _regression_checks
 
 
+def _calc_success_rate(group_membership: np.array, threshold=float):
+    group_above_threshold = group_membership[group_membership > threshold]
+    sr = group_above_threshold.size / group_membership.size
+    return sr
+
+
+def success_rate_regression(group_a, group_b, y_pred, threshold=0.50):
```

Comment:
As success rate is a term for classification, I think you can have a unique success rate function in _classification.py script and validate the type of input and threshold argument, what do you think?. If it is  only an internal function, i think it would be ok.

Reply:
So you are suggesting that the success_rate function should take the place of success_rate_regression and have different implementations based on the type of input, e.g. whether y_pred is continuous? That's a reasonable suggestion, but the reason I included it in regression is that I think it's a common use-case of regression (e.g. recruitment).